YDotNet/Document/Types/Events/EventBranch.cs

```diff
@@ -58,39 +59,35 @@ internal EventBranch(NativeWithHandle<EventBranchNative> native, Doc doc)
     /// <exception cref="YDotNetException">Value is not a <see cref="Doc" />.</exception>
     public XmlTextEvent XmlTextEvent => GetValue<XmlTextEvent>(EventBranchTag.XmlText);
 
+    /// <summary>
+    ///     Gets the <see cref="XmlFragmentEvent" /> value.
+    /// </summary>
+    /// <exception cref="YDotNetException">Value is not a <see cref="Doc" />.</exception>
+    public XmlFragmentEvent XmlFragmentEvent => GetValue<XmlFragmentEvent>(EventBranchTag.XmlFragment);
+
     private static object? BuildValue(NativeWithHandle<EventBranchNative> native, Doc doc, EventBranchTag tag)
     {
         var handle = native.Value.ValueHandle(native.Handle);
 
-        switch (tag)
+        return tag switch
```

Comment:
Do you remember why we do not use the type cache here? We should probably always use the cache or not at all.

Reply:
Not sure, IIRC this method was originally by you previously.

Reply:
Ok, weird :D